setup.py

```diff
@@ -52,6 +52,15 @@
     zip_safe=False,
     install_requires=[],  # we use conda build recipe
     entry_points=dict(
-        console_scripts=['msui = mslib.msui.msui:main'],
+        console_scripts=[
+            "mscolab = mslib.mscolab.mscolab:main",
+            "mswms = mslib.mswms.mswms:main",
+            "mswms_demodata = mslib.mswms.demodata:main",
+        ],
+        gui_scripts=[
```

Comment:
The difference between console_scripts and gui_scripts only affects Windows systems. console_scripts are wrapped in a console executable, so they are attached to a console and can use [sys.stdin](https://docs.python.org/3/library/sys.html#sys.stdin), [sys.stdout](https://docs.python.org/3/library/sys.html#sys.stdout) and [sys.stderr](https://docs.python.org/3/library/sys.html#sys.stderr) for input and output. gui_scripts are wrapped in a GUI executable, so they can be started without a console, but cannot use standard streams unless application code redirects them. Other platforms do not have the same distinction.

We maybe should not do that extra for windows. Also we have some argparse arguments for msui  and click on mssautoplot.

There is also a windows problem which needs on some configurations a terminal start from msui.

Reply:
https://github.com/Open-MSS/MSS/issues/2261

Reply:
> We maybe should not do that extra for windows. Also we have some argparse arguments for msui and click on mssautoplot.

Commandline arguments are unrelated to stdin/stdout/stderr so AFAIU they should work perfectly fine with gui_scripts as well.

But I can move them all into console_scripts as well.